katas/6kyu/detect-pangram.js

```diff
@@ -0,0 +1,5 @@
+function isPangram(string) {
+    const abc = new Set(['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']);
```

Comment:
Set is handy stuff to replace native `array` when you want to use `set`, `has` or `get` functions. also to avoid duplicated items in the "array".
but in your case, it's just alphabet without duplicates and only thing you need here is `size` that equals `length` for `string` or `array`.
So, it's enough to have just
```js
const alphabet = 'abcdefghijklmnopqrstuvwxyz'.split('')
```
to achieve the same result